introduction.md

```diff
@@ -1,6 +1,6 @@
 Hey, my name is Elayna. I'm a student. My favorite Code School path is Javascript!
 
 My deadly skills include: 
-*Mastering Git
-*Mastering Javascript
-*Enjoying Life
+* Mastering Git
+* Mastering Javascript
+* Enjoying Life
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?